### DISCUSSION TITLE
SOLVED: QlikSense 2021 May Support with Version 5.6.2?

### DISCUSSION THREAD

**Discussion by USBWAG** [ptarmiganlabs/butler-sos#176] [Q&A]
Hello Together

I have been following your work a bit, and find your stuff really great for the QlikSense world. 
Now I tried your SOS Butler, worked fine with QS version 2019 April, but not anymore with version 2021 May, do you know anything about incompatibility?

**Comment by mountaindude** [comment]
There are a couple of things to consider here: 
- Different versions of Sense. The Sense features used by Butler SOS have been pretty stable over the years, so this is not a likely source of errors - but still a possibility.  
- DIfferent versions of Butler SOS. Lots of features have been added over the past years, with associated changes in for example the config file.

With the above in mind: if you have a Butler SOS instance that works with 2019 April, then reconfigure it (change host name, new certificates etc), it should work also with 2021 May.
Btw, I strongly recommend that you use the latest Butler SOS version. There are bug fixes, security patches and new features in all releases.

Easiest is probably to look at the logs. 
Can you share what things look like when you start Butler SOS configured for the May 2021 server?
If needed you can anonymise the logs if they contain sensitive data.

**Reply by USBWAG** [reply]
Hello, Sorry was on vacation.
But I'll take care of that and you like logs etc.. Provide. Will certainly find time this week.

**Reply by mountaindude** [reply]
...and I will test Butler SOS against May 2021 once summer vacations are over.
While unlikely it's of course possible that some Sense API has changed in the latest version

**Reply by mountaindude** [reply]
@USBWAG, I have now tested Butler 5.6.2 with Qlik Sense Enterprise on Windows 2021-May patch2 and it works fine here.

How are you running Butler SOS? Native Node.js app or as Docker container?
Can you share an (anonymised) config file?

**Reply by USBWAG** [reply]
Hi Göran, 

Because I wanted to test if we can use it I installed it as Native Node.js APP on my device.
However, it will deploy directly to a Docker environment when we deploy it. 
As I noted last week, there is currently an issue on our Qlik environment. We are currently fixing this, but until it is fixed, this ticket will be closed. Thank you very much for the help.

I will give you feedback as soon as we get a response from Qlik if it is working after that. Otherwise, I would open a new issue here with all the relevant logs confs and settings.
